src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -301,8 +301,9 @@ emit_simd_ins_for_binary_op (MonoCompile *cfg, MonoClass *klass, MonoMethodSigna
 				instc0 = OP_FADD;
 				break;
 			case SN_Divide:
-			case SN_op_Division:
-				if (strcmp ("Vector4", m_class_get_name (klass)) && strcmp ("Vector2", m_class_get_name (klass))) {
+			case SN_op_Division: {
+				const char *class_name = m_class_get_name (klass);
+				if (strcmp ("Vector2", class_name) && strcmp ("Vector4", class_name) && strcmp ("Quaternion", class_name) && strcmp ("Plane", class_name)) {
```

Comment:
Does it work for Vector2? Probably should move all the Vector2 related change to your future Vecor2/3 PR.

Reply:
This is a filter of "don't execute this code for these types" where "Vector2" was already part of the condition.

I opted to not remove it, even though it isn't actually going to be hit today, given that it was pre-existing.

Reply:
Gotcha!